### PR TITLE
Replace clip.test.js with new tests

### PR DESCRIPTION
**Pull request by KaiVolland**
Transforms the old rendering test for layer clipping to the new rendering test approach.

### CODE REVIEW

File: rendering/cases/layer-clipping/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+]).transform('EPSG:4326', 'EPSG:3857');
+
+const style = new Style({
+  stroke: new Stroke({
+    width: 2,
+    color: 'blue'
+  })
+});
+
+layer.on('precompose', function(event) {
```
Comment (ahocevar) [inline rendering/cases/layer-clipping/main.js]: There are no 'precompose' and 'postcompose' events any more. Use 'prerender' and 'postrender' instead.

Comment (ahocevar) [inline rendering/cases/layer-clipping/main.js]:
```suggestion
layer.on('prerender', function(event) {
```

File: rendering/cases/layer-clipping/main.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  context.save();
+
+  const vectorContext = event.vectorContext;
+  vectorContext.setStyle(style);
+  vectorContext.drawGeometry(geometry);
+
+  context.clip();
+});
+
+layer.on('postcompose', function(event) {
```
Comment (ahocevar) [inline rendering/cases/layer-clipping/main.js]:
```suggestion
layer.on('postrender', function(event) {
```

**Comment by KaiVolland** [comment]
Adapted the code and force pushed. Seems to be good now.

**Pull request merged**